<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#131239;</string>
  </property>
  <widget class="QRadioButton" name="e_level1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 14pt &quot;Georgia&quot;;
background:white;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>Level 1</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Images and icon/Unlocked.png</normaloff>../Images and icon/Unlocked.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="e_level2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>150</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 14pt &quot;Georgia&quot;;
background:white;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>Level 2</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Images and icon/Locked.png</normaloff>../Images and icon/Locked.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Easy mode:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="e_level3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>150</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 14pt &quot;Georgia&quot;;
background:white;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>Level 3</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Images and icon/Locked.png</normaloff>../Images and icon/Locked.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_level1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 14pt &quot;Georgia&quot;;
background:white;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>Level 1</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Images and icon/Locked.png</normaloff>../Images and icon/Locked.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_level2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>210</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 14pt &quot;Georgia&quot;;
background:white;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>Level 2</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Images and icon/Locked.png</normaloff>../Images and icon/Locked.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_level3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>210</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 14pt &quot;Georgia&quot;;
background:white;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>Level 3</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Images and icon/Locked.png</normaloff>../Images and icon/Locked.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="h_level1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 14pt &quot;Georgia&quot;;
background:white;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>Level 1</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Images and icon/Locked.png</normaloff>../Images and icon/Locked.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="h_level2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 14pt &quot;Georgia&quot;;
background:white;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>Level 2</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Images and icon/Locked.png</normaloff>../Images and icon/Locked.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="h_level3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>270</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 14pt &quot;Georgia&quot;;
background:white;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>Level 3</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Images and icon/Locked.png</normaloff>../Images and icon/Locked.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>191</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Medium mode:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>161</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Georgia&quot;;
color:white;
</string>
   </property>
   <property name="text">
    <string>Hard mode:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Select Level</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="f_back">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 italic 14pt &quot;Georgia&quot;;
background:white;
border-radius:5px;
color:black;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Images and icon/Back.png</normaloff>../Images and icon/Back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Select Skin</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="skin_1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Mallet images/1.png</normaloff>../Mallet images/1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="skin_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>420</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Mallet images/2.png</normaloff>../Mallet images/2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="skin_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>420</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Mallet images/8.png</normaloff>../Mallet images/8.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>470</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Unlocked</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>470</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Unlocks at level 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>470</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Unlocks at level 3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="skin_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>520</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Mallet images/6.png</normaloff>../Mallet images/6.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="skin_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>520</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Mallet images/5.png</normaloff>../Mallet images/5.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="skin_6">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>520</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Mallet images/7.png</normaloff>../Mallet images/7.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="skin_7">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>620</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Mallet images/4.png</normaloff>../Mallet images/4.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="skin_8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>620</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Mallet images/9.png</normaloff>../Mallet images/9.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="skin_9">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>620</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Mallet images/3.png</normaloff>../Mallet images/3.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>570</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Unlocks at level 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>570</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Unlocks at level 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>570</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Unlocks at level 3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>670</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Unlocks at level 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>670</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Unlocks at level 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>670</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Unlocks at level 3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="f_play">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>720</y>
     <width>150</width>
     <height>65</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Arial Rounded MT Bold&quot;;
padding:15px;
background-color:green;
color:white;

</string>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
